File: 24_C#Fundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_01_Lecture/_09_SumOfOddNumbers/_09_SumOfOddNumbers.cs
using System;

public class _09_SumOfOddNumbers
{
    public static void Main(string[] args)
    {
        // Read input value
        int n = int.Parse(Console.ReadLine());

        // Add variable
        int sum = 0;

        // Sum odd numbers
        for (int i = 1; i < n * 2; i += 2)
        {
            sum += i;
            Console.WriteLine(i);
        }

        // Print Result
        Console.WriteLine("Sum: " + sum);
    }
}

File: 02_ProgrammingFundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_01_Lecture/_09_SumOfOddNumbers.java
package _02_ProgrammingFundamentals._01_BasicSyntaxConditionalStatementsAndLoops._01_Lecture;

import java.util.Scanner;

public class _09_SumOfOddNumbers {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n = Integer.parseInt(scan.nextLine());

        // Add variable
        int sum = 0;
        
        // Sum odd numbers
        for (int i = 1; i < n * 2; i += 2) {
            sum += i;
            System.out.println(i);
        }
        
        // Print result
        System.out.println("Sum: " + sum);

        scan.close();
    }
}
